ivy.el

```diff
@@ -4510,6 +4510,9 @@ Otherwise, forward to `ivy-kill-line'."
    ("j"
     ivy--switch-buffer-other-window-action
     "other window")
+   ("t"
+    switch-to-buffer-other-tab
```

Comment:
Should this not also support the extra stuff that `ivy--switch-buffer-other-window-action` (phew) does in the "other window" case?

Reply:
Done. Implemented `ivy--switch-buffer-other-tab-action` similarly, and used it where appropriate.